src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -108,7 +108,7 @@
   <Target Name="PreCloudBuild">
     <!-- copy runner scripts so they can be uploaded as supplemental payload -->
     <ItemGroup>
-      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.py" />
+      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.*" />
```

Comment:
As of now I believe there are roughly 4 files and I am adding another, subject to more number of files based on any specific operations we would like to perform based on perf, func, stress, etc. Adding each file by name therefore sounds a little risky. Also, I am modifying *.py to ensure the .sh file is also included. Is there a cleaner way to accomplish this.